File: mobile-client/android/app/src/main/java/app/mobiling/client/dashboard/DashboardRouteMap.kt
package app.mobiling.client.dashboard
class DashboardRouteMap {
    fun primarySections(): List<DashboardPrimarySection> = listOf(
        DashboardPrimarySection.CATALOG,
        DashboardPrimarySection.MESSAGE,
        DashboardPrimarySection.VENDOR,
    )
    fun entryFlows(): List<DashboardEntryFlow> = listOf(DashboardEntryFlow.AUTH)
    fun navigationOwners(): Map<String, String> = mapOf(
        "catalog" to "catalog/navigation",
        "vendor" to "vendor/navigation",
        "order" to "order/ownership",
    )
}

File: client/ios/App/Dashboard/DashboardRouteMap.swift
import Foundation
public struct DashboardRouteMap: Sendable {
    public init() {}
    public func primarySections() -> [DashboardPrimarySection] { [.catalog, .message, .vendor] }
    public func entryFlows() -> [DashboardEntryFlow] { [.auth] }
    public func navigationOwners() -> [String: String] {
        ["catalog": "catalog/navigation", "vendor": "vendor/navigation", "order": "order/ownership"]
    }
}
